Sources/MongoMobile/MongoMobile.swift

```diff
@@ -75,27 +88,30 @@ public class MongoMobile {
      * Perform required operations to initialize the embedded server.
      * 
      * Parameters:
-     *  - logger: An optional `MongoMobileLogger` to use with the embedded server.
-     *            This logger will be used across all DBs and clients until the 
-     *            embedded server is shut down via `close()`.
+     *  - options: options to set on the embedded server.        
      *
      * Throws:
      *  - `MongoMobileError.invalidInstance` if there is any error initializing 
      *    the embedded server.
      */
-    public static func initialize(withLogger logger: MongoMobileLogger? = nil) throws {
+    public static func initialize(_ options: MongoMobileOptions? = nil) throws {
```

Comment:
[nit] in the driver, we never omit the options keyword arg. For consistency, we may want to keep it here too, even though it's the only argument.

Reply:
good point, updated.